tabpy/tabpy_server/handlers/__init__.py

```diff
@@ -4,10 +4,10 @@
 
 from tabpy.tabpy_server.handlers.endpoint_handler import EndpointHandler
 from tabpy.tabpy_server.handlers.endpoints_handler import EndpointsHandler
-from tabpy.tabpy_server.handlers.evaluation_plane_handler\
-    import EvaluationPlaneHandler
+from tabpy.tabpy_server.handlers.evaluation_plane_handler import EvaluationPlaneHandler
 from tabpy.tabpy_server.handlers.query_plane_handler import QueryPlaneHandler
 from tabpy.tabpy_server.handlers.service_info_handler import ServiceInfoHandler
 from tabpy.tabpy_server.handlers.status_handler import StatusHandler
-from tabpy.tabpy_server.handlers.upload_destination_handler\
-    import UploadDestinationHandler
+from tabpy.tabpy_server.handlers.upload_destination_handler import (
```

Comment:
Seems like () and ,  can be removed

Reply:
Unfortunately can't do that without exceeding the 88 character line limit, so black makes this a multiline import